Packs/DarktraceASM/pack_metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "DarktraceASM",
+    "description": "Populates Darktrace ASM Risks in Cortex XSOAR, allowing for cross-platform automated investigation and response.",
+    "support": "partner",
+    "currentVersion": "1.0.1",
```

Comment:
@maimorag - should this be just `1.0.0` as this is a new pack?

Reply:
@seanmacdonald8 Please update the currentVersion to **1.0.0**.